wagtailio/project_styleguide/templates/patterns/components/streamfields/rich_text_block/rich_text_block.html

```diff
@@ -1,5 +1,7 @@
 {% load wagtailcore_tags %}
 
-<div>
-    {{ value.rich_text }}
+<div class="grid">
+    <div class="rich-text rich-text--sf">
+        {{ value.rich_text }}
```

Comment:
Question: Do we need the `|richtext` filter on here?

Reply:
I think we do! : D